homeassistant/components/hassio/config_flow.py

```diff
@@ -19,6 +19,6 @@ async def async_step_system(
     ) -> ConfigFlowResult:
         """Handle the initial step."""
         # We only need one Hass.io config entry
-        await self.async_set_unique_id(DOMAIN)
-        self._abort_if_unique_id_configured()
+        if self._async_current_entries():
```

Comment:
Should we declare this in the manifest instead?

https://developers.home-assistant.io/docs/creating_integration_manifest#single-config-entry-only

Reply:
does this also works for `system` config flow steps?

Reply:
Yes, it should work.

Reply:
nice 👍 so we could also make use of it in the other system integrations (_in separate PRs_)?